patches/extra/ungoogled-chromium/add-gruvbox-distilled.patch

```diff
@@ -0,0 +1,300 @@
+@@ -69,7 +69,7 @@ function setTextDirection(direction) {
+ }
+ 
+ // These classes must agree with the font classes in distilledpage.css.
+-const themeClasses = ['light', 'dark', 'sepia'];
++const themeClasses = ['light', 'sepia', 'gruv', 'dark'];
+ const fontFamilyClasses = ['sans-serif', 'serif', 'monospace'];
+ 
+ // Get the currently applied appearance setting.
+@@ -89,12 +89,14 @@ function useFontFamily(fontFamily) {
+ 
+ function updateToolbarColor(theme) {
+   let toolbarColor;
+-  if (theme === 'sepia') {
++  if (theme === 'light') {
++    toolbarColor = '#F5F5F5';
++  } else if (theme === 'sepia') {
+     toolbarColor = '#BF9A73';
+-  } else if (theme === 'dark') {
+-    toolbarColor = '#1A1A1A';
++  } else if (theme === 'gruv') {
++    toolbarColor = '#282828';
+   } else {
+-    toolbarColor = '#F5F5F5';
++    toolbarColor = '#1A1A1A';
+   }
+   $('theme-color').content = toolbarColor;
+ }
+--- a/components/dom_distiller/core/mojom/distilled_page_prefs.mojom
++++ b/components/dom_distiller/core/mojom/distilled_page_prefs.mojom
+@@ -6,8 +6,9 @@ module dom_distiller.mojom;
+ 
+ enum Theme {
+   kLight = 0,
+-  kDark = 1,
+-  kSepia = 2,
++  kSepia = 1,
++  kGruv = 2,
++  kDark = 3,
+ };
+ 
+ enum FontFamily {
+--- a/components/dom_distiller/core/viewer.cc
++++ b/components/dom_distiller/core/viewer.cc
+@@ -36,14 +36,16 @@ namespace viewer {
+ namespace {
+ 
+ // JS Themes. Must agree with useTheme() in dom_distiller_viewer.js.
+-const char kDarkJsTheme[] = "dark";
+ const char kLightJsTheme[] = "light";
+ const char kSepiaJsTheme[] = "sepia";
++const char kGruvJsTheme[] = "gruv";
++const char kDarkJsTheme[] = "dark";
+ 
+ // CSS Theme classes.  Must agree with classes in distilledpage.css.
+-const char kDarkCssClass[] = "dark";
+ const char kLightCssClass[] = "light";
+ const char kSepiaCssClass[] = "sepia";
++const char kGruvCssClass[] = "gruv";
++const char kDarkCssClass[] = "dark";
+ 
+ // JS FontFamilies. Must agree with useFontFamily() in dom_distiller_viewer.js.
+ const char kSerifJsFontFamily[] = "serif";
+@@ -66,20 +68,24 @@ std::string GetPlatformSpecificCss() {
+ 
+ // Maps themes to JS themes.
+ const std::string GetJsTheme(mojom::Theme theme) {
+-  if (theme == mojom::Theme::kDark)
+-    return kDarkJsTheme;
++  if (theme == mojom::Theme::kLight)
++    return kLightJsTheme;
+   if (theme == mojom::Theme::kSepia)
+     return kSepiaJsTheme;
+-  return kLightJsTheme;
++  if (theme == mojom::Theme::kGruv)
++    return kGruvJsTheme;
++  return kDarkJsTheme;
+ }
+ 
+ // Maps themes to CSS classes.
+ const std::string GetThemeCssClass(mojom::Theme theme) {
+-  if (theme == mojom::Theme::kDark)
+-    return kDarkCssClass;
++  if (theme == mojom::Theme::kLight)
++    return kLightCssClass;
+   if (theme == mojom::Theme::kSepia)
+     return kSepiaCssClass;
+-  return kLightCssClass;
++  if (theme == mojom::Theme::kGruv)
++    return kGruvCssClass;
++  return kDarkCssClass;
```

Comment:
I'm wondering: why does the Dark and Light CSS classes need to be rearranged here?

Reply:
I sorted it, so the order stays the same throughout the code. Light, Sepia, Gruv, Dark